front/src/common/redux/services/sudentRegistrationService.js

```diff
@@ -0,0 +1,18 @@
+async function studentRegistrationService(student) {
+    const url = 'http://127.0.0.1:5000/students';
+    const body = JSON.stringify(student.payload);
+    const options = {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body
+    }
+    const response = await fetch(url, options);
```

Comment:
Please, use `axios` library for making HTTP requests. Also, create a separate file where you configure default `axios` instance and don't hardcode URL of the backend in the string because url can change. Instead take it from the environment variable